nucypher/network/middleware.py

```diff
@@ -230,12 +223,16 @@ def reencrypt(self, ursula: 'Ursula', reencryption_request_bytes: bytes):
 
     def check_rest_availability(self, initiator, responder):
         response = self.client.post(node_or_sprout=responder,
-                                    data=bytes(initiator.metadata()),
-                                    path="ping",
+                                    data=bytes(initiator),
```

Comment:
Does that really work? `initiator` is an `Ursula`, and it doesn't have `__bytes__`

Reply:
No, this won't work and needs to use the `NodeMetadata` API.  At least this endpoint is meant for use with the `AvailabilityTracker` which is disabled currently.